app/Http/Controllers/Widgets/TopInterfacesController.php

```diff
@@ -54,11 +55,15 @@ public function getView(Request $request)
             ->select('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
             ->groupBy('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
             ->where('poll_time', '>', Carbon::now()->subMinutes($data['time_interval'])->timestamp)
+            ->where([['ignore', '=', 0], ['disabled', '=', 0]])
```

Comment:
Use the scope. https://github.com/librenms/librenms/blob/master/app/Models/Port.php#L163

Reply:
how do i use it?

Reply:
https://laravel.com/docs/7.x/eloquent#query-scopes

Reply:
changed :)